about.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>African-women-empowerment</title>
+    <!-- Link to COCOGOOSE and Lato Font Family-->
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link href="http://fonts.googleapis.com/css?family=Lato:400,700" rel="stylesheet" type="text/css" />
+    <!-- Link to the Font-Awesome Font -->
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200&display=swap" rel="stylesheet" />
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.2/css/font-awesome.min.css" />
+
+    <!-- CSS only -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor" crossorigin="anonymous" />
+    <link rel="stylesheet" href="css/style.css" />
+    <!-- JavaScript Bundle with Popper -->
+    <script defer src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2" crossorigin="anonymous"></script>
+    <script defer src="js/menu.js"></script>
+    <script defer src="js/about.js"></script>
+</head>
+
+<body>
+    <header class="nav-container">
+        <div class="container-fluid header1">
+            <ul class="row">
+                <li class="col-md-6"></li>
+                <li class="col-md-1">
+                    <a href="#"><i class="fa fa-facebook"></i></a>
+                </li>
+                <li class="col-md-1">
+                    <a href="# "><i class="fa fa-twitter"></i></a>
+                </li>
+                <li class="col-md-1">English</li>
+                <li class="col-md-1">My Page</li>
+                <li class="col-md-1">Logoout</li>
+                <li class="col-md-1"></li>
+            </ul>
+        </div>
+        <div class="container-fluid header2">
+            <i class="mt-5 hamburger-menu openMenu fa fa-bars"></i>
+            <ul class="row header2List mainMenu">
```

Comment:
Your mobile menu works seamlessly, however, the background picture it too opaque which makes the menu item on the mobile menu not to standout clearly. You have two options here:
- [ ] You can make it's background transparent, or
- [ ]  Make it the same color/pattern as the top section background